src/module/remove.js

```diff
@@ -0,0 +1,14 @@
+import { getTodos, sortList, savedTodos } from './storage.js';
+
+export default function removeTask(idToDelete) {
+  let todos = getTodos();
+  todos = todos.filter((todo) => {
+    if (JSON.stringify(todo.index) === idToDelete) {
+      return false;
+    }
```

Comment:
To make your function clean you can use the `arrow` instead of the regular function.

Reply:
Thank you. I will correct it.